### ISSUE TITLE
`rextendr::document` failes if rextendr is not installed (but loaded by `devtools::load_all`)

### ISSUE THREAD

**Issue by eitsupi** [extendr/rextendr#295]
I have encountered this many times during my development work and it always seems to fail in the following place.
https://github.com/extendr/rextendr/blob/1a43843b191536ab057949b1c40ca3e9e9949b90/R/register_extendr.R#L90-L107

It seems that `make_wrappers_externally` is not working, but I have not been able to determine the cause.

**Comment by JosiahParry** [comment]
Do you have a reprex? Would this bug _only_ affect folks who are developing rextendr? Normal folks who would `install.packages()` would not be affected, right?

**Comment by eitsupi** [comment]
> Do you have a reprex? Would this bug _only_ affect folks who are developing rextendr? Normal folks who would `install.packages()` would not be affected, right?

Yes, it always reproduces in my environment when I run `devtools::test()` after running `devtools::load_all()` on Ubuntu.
